integration-tests/src/test/java/org/openmrs/maven/plugins/CreateProjectIT.java

```diff
@@ -65,8 +65,11 @@ public void createProject_shouldCreatePlatformModuleProject() throws Exception{
         addTaskParam("moduleDescription", "none");
         addTaskParam("groupId", "org.openmrs.module");
         addTaskParam("version", "1.0.0-SNAPSHOT");
+        addTaskParam("junitVersion", "4.13.2");
+        addTaskParam("mockitoVersion", "3.12.4");
+        addTaskParam("junitJupiterVersion", "5.11.3");
 
-        addAnswer("1.11.6");
+        addAnswer("2.7.0");
```

Comment:
Any specific reason why you changed this to 2.7?

Reply:
removed `1.11.6` since the least version we are supporting is `2.5.0` though no specific reason why i choose `2.7.0` and not `2.5.0` or `2.6.0`